extensions/positron-r/package.nls.json

```diff
@@ -34,10 +34,10 @@
 	"r.configuration.title": "R",
 	"r.configuration.title-dev": "Advanced",
-	"r.configuration.customRootFolders.markdownDescription": "List of additional folders to search for R installations. These folders are searched after and in the same way as the default folder for your operating system (e.g. `C:/Program Files/R` on Windows).",
-	"r.configuration.customBinaries.markdownDescription": "List of additional R binaries. If you want to use an R installation that is not automatically discovered, provide the path to its binary here. For example, on Windows this might look like `C:/some/unusual/location/R-4.4.1/bin/x64/R.exe`.",
-	"r.configuration.interpreters.exclude.markdownDescription": "List of absolute paths to R binaries or folders containing R binaries to exclude from the available R installations. These interpreters will not be displayed in the Positron UI.\n\nExample: On Linux, add `/custom/location/R/4.3.0/bin/R` to exclude the specific binary, or `/custom/location` to exclude any R installations within the directory.\n\nRequires a restart to take effect.",
-	"r.configuration.interpreters.override.markdownDescription": "List of absolute paths to R binaries or folders containing R binaries to override the list of the available R installations. Only the interpreters found at the specified paths will be displayed in the Positron UI.\n\nThis setting takes precedence over the `#positron.r.interpreters.exclude#`, `#positron.r.customBinaries#` and `#positron.r.customRootFolders#` settings.\n\nExample: On Linux or Mac, add `/custom/location/R/4.3.0/bin/R` to include only this specific installation, or `/custom/location` to include only R installations found within the directory.\n\nRequires a restart to take effect.",
+	"r.configuration.customRootFolders.markdownDescription": "List of absolute paths to additional folders to search for R installations. These folders are searched after and in the same way as the default folder for your operating system (e.g. `C:/Program Files/R` on Windows).",
+	"r.configuration.customBinaries.markdownDescription": "List of absolute paths to additional R binaries. If you want to use an R installation that is not automatically discovered, provide the path to its binary here. For example, on Windows this might look like `C:/some/unusual/location/R-4.4.1/bin/x64/R.exe`.",
+	"r.configuration.interpreters.exclude.markdownDescription": "List of absolute paths to R binaries or folders containing R installations to exclude from the available R interpreters. These interpreters will not be displayed in the Positron UI.\n\nIf an interpreter is both included via `#positron.r.customRootFolders#` or `#positron.r.customBinaries#` and excluded via `#python.interpreters.exclude#`, it will not be displayed in the Positron UI.\n\nExample: On Linux, add `/custom/location/R/4.3.0/bin/R` to exclude the specific binary, or `/custom/location` to exclude any R installations within the directory.\n\nRequires a restart to take effect.",
```

Comment:
Leftover reference to python, instead of R in `#python.interpreters.exclude`

Reply:
thanks for catching this!